### DISCUSSION TITLE
سایت linkfars.com/reportage/

### DISCUSSION THREAD

**Discussion by chromer030** [MasterKia/PersianBlocker#64] [General]
این سایت مث اینکه ریپورتاژ انجام میده.

**Comment by MasterKia** [comment]
https://github.com/MasterKia/PersianBlocker/commit/eb0ccf1b77b4f5d2cf216eca2e348fb14059f66a
https://github.com/MasterKia/PersianBlocker/commit/45737020e5c2832e35b4d32cbfa131fe8c6ffed1

اگه میتونید یک نمونه بفرستید که از این سایت در سایتی استفاده شده باشه.

**Reply by chromer030** [reply]
توی همین لینک یه جدولی هست که لیست سایت ها رو نوشته : linkfars.com/reportage/

**Reply by MasterKia** [reply]
بسیار خوب، من لینک رو باز نکرده بودم مستقیم خود سایت رو دیدم.
